README.md

```diff
@@ -1,2 +1,7 @@
 # cookie-stand
 week 02 lab - cookie stand
+
+Google fonts utilized:
+-Cormorant Garamond
```

Comment:
Pro MarkDown tip: you need a space between this dash and the words to make the bullet points show up as expected!